### ISSUE TITLE
Errant fund not aligning under middle 'Fund' column

### ISSUE THREAD

**Issue by nelsonri72** [codeisland/RI-Budget-Sankey#6]
Fund 'RI Higher Education Assistance Auth' not aligning under middle 'Fund' column

![rogue fund not aligning under middle fund column](https://cloud.githubusercontent.com/assets/7595000/6173261/88c00f1e-b2b4-11e4-8c34-20cfeb777c56.JPG)

**Comment by nelsonri72** [comment]
![fund to fund conflict](https://cloud.githubusercontent.com/assets/7595000/6180479/6f2d6de6-b2f3-11e4-9f49-5bbe56e04c79.JPG)
Data got fixed and should resolve the issue.

Action needed: Reload fixed json and generate updated Sankey:
https://drive.google.com/open?id=0B9WwsVAN_QF5flg3T1U0TDNnRzRFNWQwbXAwRVpJM1lyY1VURzhhc21sRzFsWTZYYXhSRUk&authuser=0

(CSV file may be used as well if json doesn't work out.)

Json data file was fixed:
Before:{source:"General",target:"RI Higher Education Assistance Auth",value:45e5}

After:{source:"General",target:"RI Higher Education Assistance Auth-Agency",value:45e5}

Reason: As Fund and Agency have same name, "-Agency" was appended to missed record.  This appears to be the cause.

"Ri" also got fixed to "RI" in both data files

json source:
https://raw.githubusercontent.com/codeisland/RI-Budget-Sankey/773ce62325929452afad77ff59488e35c4703119/js/new_sankey.js

cc @jamesrhaley

**Comment by jamesrhaley** [comment]
Hi Nelson

It is literally just a strange bug in the sankey plug in.  I changed the
name to check if that had something to do with it.  I also altered the
final fund to make the money coming in and out of the final account
balance.  The sankey still turned out the same. It is an issues on my todo
list after I finish a couple of classes.  The plug in needs a new algorithm
that works all of the time.  Why our data seems to be the only case of this
issue I don't know, but it is.

 Just to make sure we are on the same page

```
General Revenue, Federal Funds, and Other Funds funnel into RI Higher
```

Education Assistance Auth

```
 then

 RI Higher Education Assistance Auth funnels into RI Higher Education
```

Assistance Auth-Agency

There is just a lot wrong with the plug in.  I have made a couple of small
tweaks so the accounts columns could rank from high value to lowest without
error.

If it is an issue that requires current attention again we can fork the
original and throw the question out to the group.  I know exactly where the
issue is in the code.

James
